sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorConverter.cs

```diff
@@ -58,6 +61,38 @@ internal static List<TelemetryItem> Convert(Batch<Activity> batchActivity, strin
             return telemetryItems;
         }
 
+        internal static List<TelemetryItem> Convert(Batch<Metric> batch, string roleName, string roleInstance, string instrumentationKey)
+        {
+            List<TelemetryItem> telemetryItems = new List<TelemetryItem>();
+            TelemetryItem telemetryItem;
+
+            foreach (var metric in batch)
+            {
+                if (metric.MetricType == MetricType.DoubleSum || metric.MetricType == MetricType.DoubleGauge)
```

Comment:
could we not export Histograms? (ofcourse the sum/count part only)
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Generated/Models/MetricDataPoint.cs#L37-L45

Reply:
Should we add it? We will not be able to say we support histograms :)
I will also bring this up with other language exporters to collect their feedback as well.

Reply:
We support part of it.. So something like "histograms gets exported without the buckets" or something.
(Not blocking this PR, just something to keep in mind as this evolves).